main.go

```diff
@@ -1,15 +1,50 @@
 package main
 
 import (
+	"blood-for-life-backend/store"
 	"net/http"
 
+	"github.com/jmoiron/sqlx"
 	"github.com/labstack/echo"
 )
 
 func main() {
 	e := echo.New()
+
+	db, err := sqlx.Connect() // configuration needed
+
+	if err != nil {
+		panic(err)
+	}
+
+	eventStore := store.NewPGEventStore(db)
+	bind(e, eventStore)
+
+	e.Logger.Fatal(e.Start(":1323"))
+
+}
+
+func bind(e *echo.Echo, eventStore store.EventStore) {
 	e.GET("/", func(c echo.Context) error {
 		return c.String(http.StatusOK, "Hello, World!")
 	})
-	e.Logger.Fatal(e.Start(":1323"))
+
+	e.POST("/api/create", func(c echo.Context) error {
+		event := new(store.Event)
```

Comment:
You won't be able to bind events, since ID doesn't exist on the client side. We would require a different set of models for the API.

Reply:
Sorry, a different set of models for the client side or server side? Or should we refactor the Event struct in event.go?

Reply:
we probably should create a new package called, apimodels, add event.go file and add CreateEventModel struct to it, then we can use it here.

Reply:
Implemented those changes in the most recent push! I added the CreateEventModel struct into event.go and edited Create() to make use of the added struct. I also included a postgres configuration in main().